target/linux/ath79/generic/base-files/etc/uci-defaults/04_led_migration

```diff
@@ -61,7 +61,8 @@ tplink,tl-wdr4300-v1-il|\
 tplink,tl-wdr4310-v1)
 	migrate_leds "blue:=green:"
 	;;
-tplink,tl-wpa8630-v1)
+tplink,tl-wpa8630-v1|\
+tplink,tl-wpa8630p-v1)
```

Comment:
This file should not be touched, since this is a new device and won't be upgraded.